### ISSUE TITLE
[BUG] After backup target is set, it is not usable 

### ISSUE THREAD

**Issue by w13915984028** [harvester/harvester#1692]
**Describe the bug**

**To Reproduce**
1. Install a new Harvester
2. Setup a correct backup target
3. Take backup action
4. It is refused

![image](https://user-images.githubusercontent.com/31133476/146414255-b328d6e4-4636-42a0-b8f0-bea050c907a7.png)

**Expected behavior**

After backup target is setting, it should be allowed to do backup

**Support bundle**

**Environment:**
 - Harvester ISO version:  Harvester master-head of 16.12.2021
 - Underlying Infrastructure (e.g. Baremetal with Dell PowerEdge R630):

**Additional context**

**Comment by w13915984028** [comment]
@guangbochen  @FrankYang0529 
This is an partial update to PR 1656, follow the original process.
https://github.com/harvester/harvester/pull/1656

Not as our expectation, the flag still needs to be set explicitly.

Otherwise below function  `checkBackupTargetConfigured ->  harvesterv1.SettingConfigured.IsTrue(targetSetting)`   will return error.  
Alternative is to change it, but not this time.
```
pkg/api/vm/handler.go
 func (h *vmActionHandler) checkBackupTargetConfigured() error 
 
 if err == nil && harvesterv1.SettingConfigured.IsTrue(targetSetting) {   ----> not meet, and return error
 
```

**Comment by w13915984028** [comment]
BTW:  With this fix, the #1673  is verified on Harvester ISO of today.

**Comment by w13915984028** [comment]
Solved in master-head version.
